packages/server/createSSR.ts

```diff
@@ -35,9 +35,7 @@ const getHTML = () => {
       : '<script src="/static/vendors.dll.js"></script><script src="/static/app.js"></script>'
 
     finalHTML = html
-      .replace('<head>', `<head>${makeHeadGtmHtml()}`)
-      .replace('<body>', `<body>${makeBodyGtmHtml()}`)
-      .replace('</head>', `${extraHead}</head>`)
+      .replace('<head>', `<head>${extraHead}`)
```

Comment:
I had to move `${extraHead}` to the opening of the `<head>` tag because I needed the `window.__ACTION__` variables to be available right at the top of the `<head>` in `template.html` as Google Tag Manager needs to initialize ASAP